doc/internals/how-to-add-new-backend.rst

```diff
@@ -55,6 +60,10 @@ This is what a ``BackendEntrypoint`` subclass should look like:
                 return False
             return ext in {".my_format", ".my_fmt"}
 
+        description = "Use .my_format files in Xarray"
+
+        docs = "https://link_to/your_backend/documentation"
```

Comment:
```suggestion
        url = "https://link_to/your_backend/documentation"
```

Reply:
Haha - just noticed that for some reason those changes didn't save locally!